examples/complete_cv_example.py

```diff
@@ -272,7 +272,7 @@ def main():
         "--mixed_precision",
         type=str,
         default=None,
-        choices=["no", "fp16", "bf16"],
+        choices=["no", "fp16", "bf16", "fp8"],
```

Comment:
```suggestion
        choices=["no", "fp8", "fp16", "bf16"],
```
Should we sort them all alphabetically/precision-ly?

Reply:
I like the order I put, which is by GPU generation (the later in the list, the more recent you need it, Volta, Ampere, Hopper)

Reply:
Makes sense! :)